superset-frontend/src/dashboard/components/DashboardBuilder.jsx

```diff
@@ -225,58 +227,72 @@ class DashboardBuilder extends React.Component {
 
     const childIds = topLevelTabs ? topLevelTabs.children : [DASHBOARD_GRID_ID];
 
-    const barTopOffset = HEADER_HEIGHT + (topLevelTabs ? TABS_HEIGHT : 0);
+    const hideDashboardHeader =
+      getUrlParam(URL_PARAMS.standalone, 'number') === 2;
```

Comment:
Maybe make `1` and `2` a constant/enum somewhere?

```ts
enum DashboardStandaloneMode {
  hideNav = 1,
  hideNavAndTitle = 2,
}
```

Reply:
done